charts/mlrun-ce/templates/config/mlrun-model-monitoring-db-config.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if index .Values.mlrun.modelMonitoring.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mlrun-model-monitoring-db-config
+data:
+{{- if .Values.mlrun.modelMonitoring.dsn}}
+  MLRUN_MODEL_ENDPOINT_MONITORING__ENDPOINT_STORE_CONNECTION: {{ .Values.mlrun.modelMonitoring.dsn }}
+{{- else }}
+  MLRUN_MODEL_ENDPOINT_MONITORING__ENDPOINT_STORE_CONNECTION: mysql+pymysql://root@mlrun-db:3306/monitoring
+{{- end }}
```

Comment:
Maybe define a `mlrun-ce.mlrun.modelMonitoringDSN` template in the `_helpers.tpl`? See the `mlrun-ce.jupyter.mlrunUIURL` template for example